pm_muairss_write/pm_muairss_write.xml

```diff
@@ -64,10 +70,19 @@
             </when>
             <when value="false"/>
         </conditional>
+        <param type="select" name="allpos_ext" label="All Positions File" help="Optionally generate output file containing all muon sites using the specified file type." optional="true">
+            <option value="cell">CELL</option>
+            <option value="cif">CIF</option>
+            <option value="xyz">XYZ (standard)</option>
+            <option value="extxyz">XYZ (extendend)</option>
+        </param>
         <param type="hidden" name="testing" label="Test mode" value="false"/>
     </inputs>
     <outputs>
         <data label="Muonated $structure.name using $params.name" name="muonated_structures" format="zip" from_work_dir="out_zip.zip"/>
+        <data label="File containing the positions of all muon sites" name="allpos_file" from_work_dir="allpos" auto_format="true">
+            <filter>(allpos_ext)</filter>
```

Comment:
If you accept the conditional - you'll need to change this - I think it's a bit more intuitive
```suggestion
            <filter>(isallpos['allpos_sel'] == "true")</filter>
```